Gruntfile.js

```diff
@@ -275,24 +275,18 @@ module.exports = function(grunt) {
     });
 
```

Comment:
You need to include "strict mode" at the top of ES6 script that will run on node in jenkins. The build is failing because of this:

```
Loading "Gruntfile.js" tasks...ERROR
>> SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
```

After that make sure grunt on Jenkins will run it. You may need to stick to ES5 for this file unless we can get nvm updated.